ekncontent/ekncontent/eknc-domain.c

```diff
@@ -549,9 +556,10 @@ eknc_domain_get_shards (EkncDomain *self)
  * Attempts to determine if the given link corresponds to content within
  * this domain.
  *
- * Returns: (transfer full): Returns an EKN URI to that content if so, and NULL otherwise.
+ * Returns: (nullable): Returns an EKN URI to that content if so, and NULL
```

Comment:
The transfer annotation is still useful.

The appropriate way to mark `NULL` in gtk-doc is `%NULL`.

Reply:
I thought transfer full was the default for non-const types?

Reply:
Better explicit than implicit. It means the documentation will also include the transfer annotation.